modelPricing.ts

```diff
@@ -146,3 +149,16 @@ export function getModelPricing(
       return pass_cost * n_passes;
   }
 }
+
+export type MRNADiffusionGenerateParams = {
+  transform: Transforms.MRNA_DIFFUSION_GENERATE;
+  unmaskings_per_step: number;
+  model: ModelOptions.mrna_foundation;
```

Comment:
how is this pricing model taking into account `num_samples`?

Reply:
Ha the most important commit didn't go through on Friday, just added it back, [this is the implementation](https://github.com/getomni-ai/model-pricing/pull/4/files#diff-adb3a9db3e202bedd326986f54b0be24553b172203d3c88210becc8b4578c420R165).